src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+struct gss_name_t_desc_struct;
+typedef struct gss_name_t_desc_struct* gss_name_t;
```

Comment:
I removed the typedefs to pointers.   
Can you please check if this is fine now?

please note that I still need to include some typedefs.
The reason being, we use Heimdal gssapi implementation on OSX (built-in) and Mit implementation of gssapi on Linux. They have different definitions of what gss_name_t means.